package/CHANGELOG

```diff
@@ -13,7 +13,7 @@ The rules for this file:
   * release numbers follow "Semantic Versioning" http://semver.org
 
 ------------------------------------------------------------------------------
-??/??/16 kain88-de, jdetle, fiona-naughton, richardjgowers, tyler.je.reddy
+??/??/16 kain88-de, fiona-naughton, richardjgowers, tyler.je.reddy, jdetle
```

Comment:
My understanding was that names are appended in the order of most recent changes. I realize now that probably doesnt matter haha